src/Emulators/M68KEmulator.hh

```diff
@@ -99,7 +99,8 @@ public:
   static std::string disassemble_one(
       StringReader& r,
       uint32_t start_address,
-      std::map<uint32_t, bool>& branch_target_addresses);
+      std::map<uint32_t, bool>& branch_target_addresses,
+      bool& prev_was_return);

```

Comment:
Since this disassembler was written first, it doesn't have some of the abstractions of the other two... I think a refactor to create a DisassemblyState object here (like we have in X86Emulator) would be worthwhile. I'll do the refactor after this is merged, since it would be beyond the scope of this change.